### PR TITLE
Add iot-device page on /partners

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done
- Add `IoT Device Programme`on the `/partners` page
- Add `IoT device` on the secondary menu 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/partners
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Refer to this [doc](https://docs.google.com/document/d/1SwCceQv625Kdc3wAvzSFBBA3qt3CxzDmc0DAfo3LJfs/edit) and [brief](https://docs.google.com/document/d/1vOIffTaJL8LeQVj-kuc1XU9p1-SQFVvQWYZHEjcDKZ8/edit)

## Issue / Card

Fixes [#4634](https://github.com/canonical-web-and-design/web-squad/issues/4634)

## Screenshots

image

### CODE REVIEW

**Comment by minkyngkm** [comment]
Hi @pmahnke! Do you think I need to make an issue for this PR?

**Comment by minkyngkm** [comment]
This PR need to be merged after this [PR](https://github.com/canonical-web-and-design/canonical.com/pull/453) is merged first

File: templates/partners/_secondary-navigation.html [hunk 1]
```diff
@@ -12,6 +12,9 @@
         <li class="breadcrumbs__item">
           <a class="breadcrumbs__link {% if request.path|get_secondary_nav_path == 'channel-and-reseller' %}p-link--active{% else %}p-link--soft{% endif %}" href="/partners/channel-and-reseller">Channel/Reseller</a>
         </li>
+        <li class="breadcrumbs__item">
+          <a class="breadcrumbs__link {% if request.path|get_secondary_nav_path == 'channel-and-reseller' %}p-link--active{% else %}p-link--soft{% endif %}" href="/partners/iot-device">IoT device</a>
```
Comment (petesfrench) [inline templates/partners/_secondary-navigation.html:16]:
```suggestion
          <a class="breadcrumbs__link {% if request.path|get_secondary_nav_path == 'iot-device' %}p-link--active{% else %}p-link--soft{% endif %}" href="/partners/iot-device">IoT device</a>
```

**Review by petesfrench — APPROVED** [review]
Just one change, but otherwise LGTM :raised_hands:

**Pull request merged**